core/server/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1891,7 +1891,7 @@ private long getWorkerStoringFile(TachyonURI path) throws FileDoesNotExistExcept
   public synchronized FileSystemCommand workerHeartbeat(long workerId, List<Long> persistedFiles)
       throws FileDoesNotExistException, InvalidPathException, AccessControlException {
     for (long fileId : persistedFiles) {
-      setState(getPath(fileId), SetAttributeOptions.defaults().setPersisted(true));
+      setState(getPath(fileId), new SetAttributeOptions.Builder().setPersisted(true).build());
```

Comment:
the master-side options still use the builder idiom, so did this for consistency ... I would prefer to change all the master-side options to be consistent with the client-side style in a separate PR